Description: Create install dirs
 If install dirs don't exist, create them. Honor DESTDIR.
 .
 fbv (1.0b-1) unstable; urgency=low
 .
   * Initial release
Author: nick black <nick.black@sprezzatech.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: no
Reviewed-By: Nick Black <nick.black@sprezzatech.com>
Last-Update: 2012-07-07

--- fbv-1.0b.orig/Makefile
+++ fbv-1.0b/Makefile
@@ -28,6 +28,8 @@ distclean: clean
 	rm -f $(OUT) config.h
 
 install: $(OUT)
+	[ -d $(DESTDIR)/$(bindir) ] || mkdir -p $(DESTDIR)$(bindir)
+	[ -d $(DESTDIR)/$(mandir)/man1 ] || mkdir -p $(DESTDIR)$(mandir)/man1
 	cp $(OUT) $(DESTDIR)$(bindir)
 	gzip -9c $(OUT).1 > $(DESTDIR)$(mandir)/man1/$(OUT).1.gz
 
